Assets/MixedRealityToolkit/_Core/Definitions/Utilities/TrackedObjectType.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities
+{
+    public enum TrackedObjectType
+    {
+        /// <summary>
+        /// Calculates position and orientation from the main camera.
+        /// </summary>
+        Head = 0,
+        /// <summary>
+        /// Calculates position and orientation from the left motion-tracked controller.
+        /// </summary>
+        MotionControllerLeft,
```

Comment:
Maybe we should be more platform agnostic here.

`TrackedControllerLeft`?

Reply:
Microsoft, Valve, and Oculus all use "motion controller" as a generic term to refer to their devices. Seems pretty platform agnostic to me, and clearer than Tracked Controller imo.